libraries/classes/Admin.php

```diff
@@ -15,10 +15,20 @@ public static function getAllUserDetails()
       $db = new Factory;
       $mysql = $db->getDBO();
   
-      $sql  = "SELECT sum(po.status) as totalPosts,sum(us.activation) as totalUsers, from posts po INNER JOIN users us ON po.uid = us.id WHERE 1"; 
-  
+      $sql  = "SELECT sum(us.activation) as totalUsers from users us"; 
```

Comment:
Learn about joins to make this one query. 
https://www.tutorialspoint.com/sql/sql-using-joins.htm